contract/test/test-sell-concert-tickets-contract.js

```diff
@@ -134,8 +134,21 @@ const alice = async (
   const toTrade = E(publicFacet).makeTradeInvitation();
 
   const seat = E(zoe).offer(toTrade, proposal, { Price: pmt });
-  const tickets = await E(seat).getPayout('Tickets');
+  const result = await E(seat)
+    .getOfferResult()
+    .catch(err => {
+      if (successfulTrade) {
+        // We expected a successful trade, got err instead.
+        t.fail(err.message);
+      } else {
+        // We expected a unsuccessful trade
+        t.pass();
+      }
+    });
```

Comment:
I'm not sure about flow control here. Perhaps the way to do it is:

```suggestion
  const resultP = E(seat).getOfferResult();
  await (successfulTrade ? t.notThrowsAsync(resultP) : t.throwsAsync(resultP));
  if (!successfulTrade) {
    return;
  }
  const result = await resultP;
```